setup.py

```diff
@@ -6,13 +6,12 @@
 try: import numpy
 except ImportError: install_requires.append('numpy')
 
-if sys.version_info[0] > 2:
-    install_requires.append('python-dateutil>=2.0')
-else:
+install_requires.append('python-dateutil>=2.2')
```

Comment:
could you move this up to line 4?

Reply:
I'd like to keep this file the same as master (<https://github.com/pycollada/pycollada/blob/master/setup.py>) except version. Could you update?

Reply:
@jterrace I updated setup.py and sent PR (#82). Please work on it first, which does block this PR.

Reply:
@jterrace Do you have any other comments?